src/CSharpMongoMigrations/MongoClientFactory.cs

```diff
@@ -1,27 +0,0 @@
-﻿using MongoDB.Driver;
```

Comment:
This class must be restored!
Please have a look at my comment for MigrationRunner cts.